rollup.config.js

```diff
@@ -65,6 +67,12 @@ export default {
     dir: outputDir,
     format: 'es',
     hoistTransitiveImports: false,
+    intro() {
+      return `/* ${fs.readFileSync(
+        path.resolve(__dirname, './LICENSE'),
+        'utf8'
+      )} */`;
+    },
```

Comment:
It is run for the output for which it is in the output options so here for all outputs in `outputdir`. I'll make an update for that.

Reply:
The license is printed only in `vtk.js` file now. However, I did not get to do it without creating another entry/output "rule" for that.

Reply:
FYI the `vtk.js` file isn't the ESM build's entrypoint. I think it's the `index.js` file. That being said, the import strategy we have doesn't go through a single file, so we would need to consider how we want the vtk.js license to show up in downstream builds.

@finetjul would we consider a single license header in the `vtk.js` file to be sufficient? Or in each outputted .js file? We would still have the issue of preserving the license header if we output to each .js file, but we may not have any such license header if the user doesn't include `vtk.js`.

Reply:
@finetjul so currently the license is printed in the `vtk.js` file but as said by Forrest, if the user does not include the `vtk.js` file it will not be written.